tests/kernel/ltp_setup_networking.pm

```diff
@@ -79,7 +79,12 @@ EOF
     assert_script_run('sed -i \'s/^\(hosts:\s+files\s\+dns$\)/\1 myhostname/\' /etc/nsswitch.conf');
 
     foreach my $service (qw(dnsmasq finger.socket nfsserver rpcbind telnet.socket vsftpd xinetd)) {
-        assert_script_run("systemctl reenable $service");
+        if (check_var('VERSION', '12')) {
+            assert_script_run("systemctl enable $service");
```

Comment:
For better style you could use our systemctl wrapper from 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3322